app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Posts</h1>
\ No newline at end of file
+<div class="posts">
+ <%= render "shared/details", user: @user %>
+  <div>
+   <% @posts.each_with_index do |post| %>
+     <%= link_to user_post_path(user_id:@user.id, id: post.id) do%>
+       <%= render "shared/post_details", post: post %>
+        <div class="comments">
+        <% if post.comments.empty? %>
+          <p>Empty</p>
+        <% else %>
+            <% @recent_comments.each do |comment| %>
```

Comment:
- [ ] The `recent_comments` method belongs to the `Post` Model to access it you should use the following code `<% post.recent_comments.each do |comment| %>` so rails know that you want to access the model method not a local method in the current file
![Screenshot_20230309_200710](https://user-images.githubusercontent.com/25227022/224128448-5453c32a-6322-4291-8e39-b27aa6c575b0.png)